xcode/Subconscious/Shared/Components/Deck/DeckNavigationView.swift

```diff
@@ -80,6 +93,14 @@ struct DeckNavigationView: View {
             .background(
                 colorScheme == .dark ? DeckTheme.darkBg : DeckTheme.lightBg
             )
+            .toolbar {
+                MainToolbar(
+                    app: app,
+                    profileAction: {
+                        store.send(.detailStack(.requestOurProfileDetail))
+                    }
+                )
+            }
```

Comment:
Show settings icon in DeckView. We'll soon be adding the sync indicator here too.